lib/frontend/src/lib.rs

```diff
@@ -118,11 +118,11 @@
 //!     }
 //!
 //!     let flags = settings::Flags::new(settings::builder());
-//!     let res = verify_function(&func, &flags);
+//!     let mut errors = VerifierErrors::default();
+//!     let res = verify_function(&func, &flags, &mut errors);
 //!     println!("{}", func.display(None));
-//!     match res {
-//!         Ok(_) => {}
-//!         Err(err) => panic!("{}", err),
+//!     if !errors.0.is_empty() {
```

Comment:
Instead of testing errors.0.is_empty() here, it feels cleaner to test whether the return value of `verify_function` .is_err().

Reply:
Ditto. I understand that it feels a little hacky to allow an `Ok` return value without it being necessarily fine, but I thought it worked well with `?`. Furthermore, right now my objective was to introduce the new VerifierErrors type, and to use it everywhere. For now I consider (most) errors as fatal, even though some of them shouldn't halt the process completely.